### ISSUE TITLE
`ErrorMayQuit` in the kernel doesn't say how to exit the break loop

### ISSUE THREAD

**Issue by ChrisJefferson** [gap-system/gap#1652]
The GAP kernel function `ErrorMayQuit` behaves like `ErrorNoReturn` in the library -- it creates a break loop which can only be exited, not returned from.

However, it doesn't tell the user how to exit the break loop, for example (`SHIFTED_PERM` uses `ErrorMayQuit` internally)

```
gap> f := function() SHIFTED_PERM((1,2^50), 2^50); end;;
gap> f();
Error, Permutation literal exceeds maximum permutation degree -- 1125899906842624 vs 4294967295 in
  SHIFTED_PERM( (1,2 ^ 50), 2 ^ 50 ); at *stdin*:2 called from
<function "f">( <arguments> )
 called from read-eval loop at *stdin*:3
brk>
```

**Comment by wilfwilson** [comment]
I'll have a go at sorting this out.

**Comment by fingolfin** [comment]
Resolved by #1654
